### PR TITLE
Fix UI regressions related to UL default margin reset

### LINKED ISSUE #6151: Nested lists no longer indented correctly
This comes from one of the PRs that were merged into `a11y` and recently into `master`:

The most problematic place where this happens is in the event description and in minutes:

image
image

I'm not sure if one of the existing a11y PRs fixes this, but a separate PR (against `master`) for this would be appreciated because the current state is certainly broken and users will likely complain once we put that live... :/

PS: Another - minor - UI regression which is probably also caused by removing the default list margins is on the event (and category) log pages in management: The "Show...", log item list, and pagination widget are now much more squeezed together.

### PR DESCRIPTION
**Pull request by foxbunny**
- Fixed editor output
- Fixed layout of the log pages

Fixes #6151

### CODE REVIEW

**Comment by ThiefMaster** [comment]
Thanks for the quick fix! Looks already much better now! :)

I'd add some more margin though (not necessarily as much as before since that's just what I'm used to), especially because it now looks like the bullet is slightly to the left compared to a h2 title in the minutes

this pr:
image
before a11y merge:
image

File: indico/web/client/styles/editor-output.scss [hunk 1]
```diff
@@ -66,4 +66,9 @@
   code {
     background-color: #f5f5f5;
   }
+
+  ul,
+  ol {
+    margin-left: 1em;
```
Comment (ThiefMaster) [inline indico/web/client/styles/editor-output.scss:72]: 2em or 3em looks better IMHO

Comment (foxbunny) [reply indico/web/client/styles/editor-output.scss:72]: How about this?

image

Only the first level is indented 3em, the nested ones only 1.2em (aligns the text of the bullet of the nested list with the content of the parent item).

Comment (foxbunny) [reply indico/web/client/styles/editor-output.scss:72]: Tho, not sure about this. When adding the initial indent for the top-level, but we have no text preceding it, we get something like this:

image

Comment (foxbunny) [reply indico/web/client/styles/editor-output.scss:72]: Alright. Then I'll push this.

**Review by ThiefMaster — APPROVED** [review]

**Pull request merged**